packages/react/src/components/PageWizard/PageWizard.story.jsx

```diff
@@ -410,6 +410,27 @@ export const WithHorizontalProgressIndicator = () => (
 
 WithHorizontalProgressIndicator.storyName = 'With Horizontal ProgressIndicator';
 
+export const WithSpaceEqually = () => (
```

Comment:
Can we just make this a knob on an existing story? I don't think it needs a story of its own.

Reply:
Done. Removed the story and added to knob https://deploy-preview-2622--carbon-addons-iot-react.netlify.app/?path=/story/1-watson-iot-pagewizard--with-sticky-footer-stateful-example-w-validation-in-page-title-bar